content/2025/index.md

```diff
@@ -1,22 +1,34 @@
 +++
 title = "OSCW 2025: Sofia, Bulgaria"
-description = "OSCW 2025 will take place before Real World Crypto on 25 March 2025 in Sofia, Bulgaria."
+description = "OSCW 2025 will take place before Real World Crypto on 25 March 2025 at the Hilton Sofia (g.k. Lozenets, Bulgaria Blvd 1, 1421 Sofia, Bulgaria) in Sofia, Bulgaria."
```

Comment:
Can we rather attach a Google Maps link to the `Hilton Sofia` (instead of the address in paranthesis), i.e. [Hilton Sofia](https://maps.app.goo.gl/shxnwsyo2KB5o9jS7)?

Reply:
Done :)